src/org/mozilla/javascript/ClassCache.java

```diff
@@ -132,7 +101,7 @@ public synchronized void setCachingEnabled(boolean enabled) {
     }
 
     /** @return a map from classes to associated JavaMembers objects */
-    Map<CacheKey, JavaMembers> getClassCacheMap() {
+    Map<Class, JavaMembers> getClassCacheMap() {
```

Comment:
No security context -> We can revert this change https://github.com/mozilla/rhino/pull/1019